android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -292,34 +293,14 @@ public void setSupportMultipleWindows(RNCWebView view, boolean enabled){
 
   @ReactProp(name = "webViewKey")
   public void setWebViewKey(RNCWebView view, String webViewKey) {
-    Map<String, WebView> internalWebViewMap = RNCWebViewMapManager.INSTANCE.getInternalWebViewMap();
-    Map<String, RNCWebView> rncWebViewMap = RNCWebViewMapManager.INSTANCE.getRncWebViewMap();
+    view.setWebViewKey(webViewKey);
 
-    // If there is an existing RNCWebView that has an internal webview, re-attach it to this view
-    if (rncWebViewMap.containsKey(webViewKey)) {
-      RNCWebView existingView = rncWebViewMap.get(webViewKey);
-      InternalWebView existingWebView = existingView.detachWebView();
-      view.attachWebView(existingWebView);
-
-      // The chrome client was originally setup on instance creation but might be pointing to the wrong webview
-      // so it's reset here.
-      // Not entirely sure why there is a single instance of the webchrome client for all webviews?
-      setupWebChromeClient((ThemedReactContext) existingWebView.getContext(), existingWebView);
-
-    // If there is a detached internal webview attach it to this RNCWebView
-    } else if (internalWebViewMap.containsKey(webViewKey)) {
-      InternalWebView webView = (InternalWebView) internalWebViewMap.get(webViewKey);
-      view.attachWebView(webView);
+    // Remove the default webview on instance creation
+    // The actual webview will be attached
+    WebView activeWebview = RNCWebViewMapManager.INSTANCE.getInternalWebViewMap().get(webViewKey);
```

Comment:
nit: Can we name this something like `activeInternalWebView`? Looking at https://github.com/discord/react-native-webview/pull/22/files#diff-e9700631fc9fa5600d2e1ee70eb2d28aa2a1fbc695b85f99981972d3904ca485R819 , it seems that sometimes we're using the name `activeWebView` for the internal WebView, and sometimes we use it for the `RNCWebView`. That makes it a little harder for me to follow the logic sometimes

Reply:
Alternatively, we could keep this one named `activeWebView`, and we can rename the other one to `activeRncWebView`